### PR TITLE
Remove duplicate bootstrap to improve page loading speed

### PR DESCRIPTION
**Pull request by zbynek**
Summary of this pull request: don't load bootstrap twice to improve page loading speed.

### CODE REVIEW

File: views/index.hbs [hunk 1]
```diff
@@ -1,10 +1,500 @@
-<html>
-  <head>
-    <title>Jenkins Plugins List</title>
-    {{> header }}
-  </head>
-  <body>
-    <div id="grid-box"></div>
-    <script type="text/javascript" src="/main.js"></script>
-  </body>
+<!DOCTYPE html>
```
Comment (halkeye) [inline views/index.hbs:1]: this file gets overridden every 15 minutes, so hopefully you didn't actually make any changes did you?

Comment (zbynek) [reply views/index.hbs:1]: No. It's the automatically downloaded file.

**Comment by halkeye** [comment]
Your css foo is way better than mine

Does this mean the entire site is using bootstrap 3 now? and the 2-3 bits you removed were bootstrap 4?

**Review by halkeye — APPROVED** [review]

**Comment by halkeye** [comment]
I think views/index.hbs needs to be added to gitignore

**Review by timja — APPROVED** [review]

**Comment by zbynek** [comment]
Previously it was Bootstrap 4.0.0 alpha together with 4.3.* that comes from the main site, now it's just the latter.

**Review by oleg-nenashev — APPROVED** [review]
Looks right, thanks :+1:

**Pull request merged**